interface/Variables.h

```diff
@@ -1926,7 +1972,7 @@ class Variables {
     effcut_d0          =       cut_nvtx && cut_federr && hp && pt && nstrip &&       dz && pixhit && noscan && goodmod && goodroc && lx_fid && ly_fid && valmis && hitsep && trk_beta_cut;
     effcut_dz          =       cut_nvtx && cut_federr && hp && pt && nstrip && d0 &&       pixhit && noscan && goodmod && goodroc && lx_fid && ly_fid && valmis && hitsep && trk_beta_cut;
     effcut_d0_dz       =       cut_nvtx && cut_federr && hp && pt && nstrip &&             pixhit && noscan && goodmod && goodroc && lx_fid && ly_fid && valmis && hitsep && trk_beta_cut;
-    effcut_scans       =       cut_nvtx && cut_federr && hp && pt && nstrip && d0 && dz && pixhit           && goodmod            && lx_fid && ly_fid && valmis && hitsep && trk_beta_cut;
+    effcut_scans       =       cut_nvtx && cut_federr && hp && pt && nstrip && d0 && dz && pixhit           && goodmod && goodroc_scan && lx_fid && ly_fid && valmis && hitsep && trk_beta_cut;
```

Comment:
I'm not sure I understand this change correctly, is it needed just for the HV scan right? If the run is not a HV scan in line 1464 nothing should change right? because this cut affects also the Timing scan

Reply:
This is part of a more complex modification. So badroc exclusion for scans (HV and Timing) are changed to be the following:
1. Use a specific run for the exclusion defined in scan_points.h
2. If no run is defined, use the same run, but excluding the scan points when HV is changing.
So basically since for timing scan this is not relevant, it should be transparent for that. But maybe it should be checked.